spec/models/category_spec.rb

```diff
@@ -3,4 +3,49 @@
 describe Category, type: :model do
   it { should have_many(:videos) }
   it { should validate_presence_of(:name) }
+
+  describe "#recent_videos" do
+    before :each do
```

Comment:
interesting, didn't know that ```before``` worked that way.  I'll look into it a bit more.  Change implemented.